README.md

```diff
@@ -3,7 +3,9 @@ A conference call implementation using WebRTC, Socket.io and Node.js.
 
 
 # Getting Started
-Just run `npm install` and you're good to go.
+-- Run `npm install` and than 
```

Comment:
Thanks for this. I assumed people will know they have to run the app.

Please fix the typo (then, not than) and will merge.

Thank you.